tests/integration/test_k8s.py

```diff
@@ -159,3 +157,44 @@ def test_list_deployments():
             finally:
                 db_deployment.delete()
                 k8s_backend.delete_namespace(namespace)
+
+
+@pytest.mark.xfail
```

Comment:
What about adding some minikube install&set magic into [test-requirements.sh](https://github.com/user-cont/conu/blob/master/test-requirements.sh) ? Wouldn't that help ?